CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 
 ## [2.8.0] - Not released yet
 ### Added
+* [`Templates`](https://py-pdf.github.io/fpdf2/fpdf/Templates.html) can now be also defined in JSON files.
```

Comment:
```suggestion
* [`Templates`](https://py-pdf.github.io/fpdf2/fpdf/Templates.html) can now also be feed content from JSON files.
```

Reply:
That doesn't sound more correct to me (and I don't even mean the grammar... 😉 ).

In my mind, the csv/json files or python dicts contain the definition of the template. Later it may get fed custom text strings.